### ISSUE TITLE
nvmf-phy-autotest times out during nvmf_shutdown_tc1 test

### ISSUE THREAD

**Issue by mikeBashStuff** [spdk/spdk#1981]
https://ci.spdk.io/results/autotest-per-patch/builds/49182/archive/nvmf-phy-autotest/build.log

```
00:07:18.886   12:09:25	# /var/jenkins/workspace/nvmf-phy-autotest/spdk/test/bdev/bdevperf/bdevperf -r /var/tmp/bdevperf.sock --json /dev/fd/62 -q 64 -o 65536 -w verify -t 1
00:07:18.886    12:09:25	# gen_nvmf_target_json 1 2 3 4 5 6 7 8 9 10
00:07:18.886    12:09:25	# config=()
00:07:18.887    12:09:25	# local subsystem config
00:07:18.887    12:09:25	# for subsystem in "${@:-1}"
00:07:18.887    12:09:25	# config+=("$(
00:07:18.887  				cat <<- EOF
00:07:18.887  					{
00:07:18.887  					  "params": {
00:07:18.887  					    "name": "Nvme$subsystem",
00:07:18.887  					    "trtype": "$TEST_TRANSPORT",
00:07:18.887  					    "traddr": "$NVMF_FIRST_TARGET_IP",
00:07:18.887  					    "adrfam": "ipv4",
00:07:18.887  					    "trsvcid": "$NVMF_PORT",
00:07:18.887  					    "subnqn": "nqn.2016-06.io.spdk:cnode$subsystem",
00:07:18.887  					    "hostnqn": "nqn.2016-06.io.spdk:host$subsystem",
00:07:18.887  					    "hdgst": ${hdgst:-false},
00:07:18.887  					    "ddgst": ${ddgst:-false}
00:07:18.887  					  },
00:07:18.887  					  "method": "bdev_nvme_attach_controller"
00:07:18.887  					}
00:07:18.887  				EOF
00:07:18.887  			)")
00:07:18.887     12:09:25	# cat
00:07:18.890    12:09:25	# for subsystem in "${@:-1}"
00:07:18.890    12:09:25	# config+=("$(
00:07:18.890  				cat <<- EOF
00:07:18.890  					{
00:07:18.890  					  "params": {
00:07:18.890  					    "name": "Nvme$subsystem",
00:07:18.890  					    "trtype": "$TEST_TRANSPORT",
00:07:18.890  					    "traddr": "$NVMF_FIRST_TARGET_IP",
00:07:18.890  					    "adrfam": "ipv4",
00:07:18.890  					    "trsvcid": "$NVMF_PORT",
00:07:18.890  					    "subnqn": "nqn.2016-06.io.spdk:cnode$subsystem",
00:07:18.890  					    "hostnqn": "nqn.2016-06.io.spdk:host$subsystem",
00:07:18.890  					    "hdgst": ${hdgst:-false},
00:07:18.890  					    "ddgst": ${ddgst:-false}
00:07:18.890  					  },
00:07:18.890  					  "method": "bdev_nvme_attach_controller"
00:07:18.890  					}
00:07:18.890  				EOF
00:07:18.890  			)")
00:07:18.890     12:09:25	# cat
00:07:18.893    12:09:25	# for subsystem in "${@:-1}"
00:07:18.893    12:09:25	# config+=("$(
00:07:18.893  				cat <<- EOF
00:07:18.893  					{
00:07:18.893  					  "params": {
00:07:18.893  					    "name": "Nvme$subsystem",
00:07:18.893  					    "trtype": "$TEST_TRANSPORT",
00:07:18.893  					    "traddr": "$NVMF_FIRST_TARGET_IP",
00:07:18.893  					    "adrfam": "ipv4",
00:07:18.893  					    "trsvcid": "$NVMF_PORT",
00:07:18.893  					    "subnqn": "nqn.2016-06.io.spdk:cnode$subsystem",
00:07:18.893  					    "hostnqn": "nqn.2016-06.io.spdk:host$subsystem",
00:07:18.893  					    "hdgst": ${hdgst:-false},
00:07:18.893  					    "ddgst": ${ddgst:-false}
00:07:18.893  					  },
00:07:18.893  					  "method": "bdev_nvme_attach_controller"
00:07:18.893  					}
00:07:18.893  				EOF
00:07:18.893  			)")
00:07:18.893     12:09:25	# cat
00:07:18.895    12:09:25	# for subsystem in "${@:-1}"
00:07:18.895    12:09:25	# config+=("$(
00:07:18.895  				cat <<- EOF
00:07:18.895  					{
00:07:18.895  					  "params": {
00:07:18.895  					    "name": "Nvme$subsystem",
00:07:18.895  					    "trtype": "$TEST_TRANSPORT",
00:07:18.895  					    "traddr": "$NVMF_FIRST_TARGET_IP",
00:07:18.895  					    "adrfam": "ipv4",
00:07:18.895  					    "trsvcid": "$NVMF_PORT",
00:07:18.895  					    "subnqn": "nqn.2016-06.io.spdk:cnode$subsystem",
00:07:18.895  					    "hostnqn": "nqn.2016-06.io.spdk:host$subsystem",
00:07:18.895  					    "hdgst": ${hdgst:-false},
00:07:18.895  					    "ddgst": ${ddgst:-false}
00:07:18.895  					  },
00:07:18.895  					  "method": "bdev_nvme_attach_controller"
00:07:18.895  					}
00:07:18.895  				EOF
00:07:18.895  			)")
00:07:18.896     12:09:25	# cat
00:07:18.898    12:09:25	# for subsystem in "${@:-1}"
00:07:18.898    12:09:25	# config+=("$(
00:07:18.898  				cat <<- EOF
00:07:18.898  					{
00:07:18.898  					  "params": {
00:07:18.898  					    "name": "Nvme$subsystem",
00:07:18.898  					    "trtype": "$TEST_TRANSPORT",
00:07:18.898  					    "traddr": "$NVMF_FIRST_TARGET_IP",
00:07:18.898  					    "adrfam": "ipv4",
00:07:18.898  					    "trsvcid": "$NVMF_PORT",
00:07:18.898  					    "subnqn": "nqn.2016-06.io.spdk:cnode$subsystem",
00:07:18.898  					    "hostnqn": "nqn.2016-06.io.spdk:host$subsystem",
00:07:18.898  					    "hdgst": ${hdgst:-false},
00:07:18.898  					    "ddgst": ${ddgst:-false}
00:07:18.898  					  },
00:07:18.898  					  "method": "bdev_nvme_attach_controller"
00:07:18.898  					}
00:07:18.898  				EOF
00:07:18.898  			)")
00:07:18.899     12:09:25	# cat
00:07:18.901    12:09:25	# for subsystem in "${@:-1}"
00:07:18.901    12:09:25	# config+=("$(
00:07:18.901  				cat <<- EOF
00:07:18.901  					{
00:07:18.901  					  "params": {
00:07:18.901  					    "name": "Nvme$subsystem",
00:07:18.901  					    "trtype": "$TEST_TRANSPORT",
00:07:18.901  					    "traddr": "$NVMF_FIRST_TARGET_IP",
00:07:18.901  					    "adrfam": "ipv4",
00:07:18.901  					    "trsvcid": "$NVMF_PORT",
00:07:18.901  					    "subnqn": "nqn.2016-06.io.spdk:cnode$subsystem",
00:07:18.901  					    "hostnqn": "nqn.2016-06.io.spdk:host$subsystem",
00:07:18.901  					    "hdgst": ${hdgst:-false},
00:07:18.901  					    "ddgst": ${ddgst:-false}
00:07:18.901  					  },
00:07:18.901  					  "method": "bdev_nvme_attach_controller"
00:07:18.901  					}
00:07:18.901  				EOF
00:07:18.901  			)")
00:07:18.901     12:09:25	# cat
00:07:18.903    12:09:25	# for subsystem in "${@:-1}"
00:07:18.904    12:09:25	# config+=("$(
00:07:18.904  				cat <<- EOF
00:07:18.904  					{
00:07:18.904  					  "params": {
00:07:18.904  					    "name": "Nvme$subsystem",
00:07:18.904  					    "trtype": "$TEST_TRANSPORT",
00:07:18.904  					    "traddr": "$NVMF_FIRST_TARGET_IP",
00:07:18.904  					    "adrfam": "ipv4",
00:07:18.904  					    "trsvcid": "$NVMF_PORT",
00:07:18.904  					    "subnqn": "nqn.2016-06.io.spdk:cnode$subsystem",
00:07:18.904  					    "hostnqn": "nqn.2016-06.io.spdk:host$subsystem",
00:07:18.904  					    "hdgst": ${hdgst:-false},
00:07:18.904  					    "ddgst": ${ddgst:-false}
00:07:18.904  					  },
00:07:18.904  					  "method": "bdev_nvme_attach_controller"
00:07:18.904  					}
00:07:18.904  				EOF
00:07:18.904  			)")
00:07:18.904     12:09:25	# cat
00:07:18.906    12:09:25	# for subsystem in "${@:-1}"
00:07:18.906    12:09:25	# config+=("$(
00:07:18.906  				cat <<- EOF
00:07:18.906  					{
00:07:18.906  					  "params": {
00:07:18.906  					    "name": "Nvme$subsystem",
00:07:18.906  					    "trtype": "$TEST_TRANSPORT",
00:07:18.906  					    "traddr": "$NVMF_FIRST_TARGET_IP",
00:07:18.906  					    "adrfam": "ipv4",
00:07:18.906  					    "trsvcid": "$NVMF_PORT",
00:07:18.906  					    "subnqn": "nqn.2016-06.io.spdk:cnode$subsystem",
00:07:18.906  					    "hostnqn": "nqn.2016-06.io.spdk:host$subsystem",
00:07:18.906  					    "hdgst": ${hdgst:-false},
00:07:18.906  					    "ddgst": ${ddgst:-false}
00:07:18.906  					  },
00:07:18.906  					  "method": "bdev_nvme_attach_controller"
00:07:18.906  					}
00:07:18.906  				EOF
00:07:18.906  			)")
00:07:18.906     12:09:25	# cat
00:07:18.908    12:09:25	# for subsystem in "${@:-1}"
00:07:18.908    12:09:25	# config+=("$(
00:07:18.908  				cat <<- EOF
00:07:18.908  					{
00:07:18.908  					  "params": {
00:07:18.908  					    "name": "Nvme$subsystem",
00:07:18.908  					    "trtype": "$TEST_TRANSPORT",
00:07:18.908  					    "traddr": "$NVMF_FIRST_TARGET_IP",
00:07:18.908  					    "adrfam": "ipv4",
00:07:18.908  					    "trsvcid": "$NVMF_PORT",
00:07:18.908  					    "subnqn": "nqn.2016-06.io.spdk:cnode$subsystem",
00:07:18.908  					    "hostnqn": "nqn.2016-06.io.spdk:host$subsystem",
00:07:18.908  					    "hdgst": ${hdgst:-false},
00:07:18.908  					    "ddgst": ${ddgst:-false}
00:07:18.908  					  },
00:07:18.908  					  "method": "bdev_nvme_attach_controller"
00:07:18.908  					}
00:07:18.908  				EOF
00:07:18.908  			)")
00:07:18.909     12:09:25	# cat
00:07:18.911    12:09:25	# for subsystem in "${@:-1}"
00:07:18.911    12:09:25	# config+=("$(
00:07:18.911  				cat <<- EOF
00:07:18.911  					{
00:07:18.911  					  "params": {
00:07:18.911  					    "name": "Nvme$subsystem",
00:07:18.911  					    "trtype": "$TEST_TRANSPORT",
00:07:18.911  					    "traddr": "$NVMF_FIRST_TARGET_IP",
00:07:18.911  					    "adrfam": "ipv4",
00:07:18.911  					    "trsvcid": "$NVMF_PORT",
00:07:18.911  					    "subnqn": "nqn.2016-06.io.spdk:cnode$subsystem",
00:07:18.911  					    "hostnqn": "nqn.2016-06.io.spdk:host$subsystem",
00:07:18.911  					    "hdgst": ${hdgst:-false},
00:07:18.911  					    "ddgst": ${ddgst:-false}
00:07:18.911  					  },
00:07:18.911  					  "method": "bdev_nvme_attach_controller"
00:07:18.911  					}
00:07:18.911  				EOF
00:07:18.911  			)")
00:07:18.911     12:09:25	# cat
00:07:18.913    12:09:25	# jq .
00:07:18.914     12:09:25	# IFS=,
00:07:18.914     12:09:25	# printf '%s\n' '{
00:07:18.914    "params": {
00:07:18.914      "name": "Nvme1",
00:07:18.914      "trtype": "rdma",
00:07:18.914      "traddr": "192.168.100.8",
00:07:18.914      "adrfam": "ipv4",
00:07:18.914      "trsvcid": "4420",
00:07:18.914      "subnqn": "nqn.2016-06.io.spdk:cnode1",
00:07:18.914      "hostnqn": "nqn.2016-06.io.spdk:host1",
00:07:18.914      "hdgst": false,
00:07:18.914      "ddgst": false
00:07:18.914    },
00:07:18.914    "method": "bdev_nvme_attach_controller"
00:07:18.914  },{
00:07:18.914    "params": {
00:07:18.914      "name": "Nvme2",
00:07:18.914      "trtype": "rdma",
00:07:18.914      "traddr": "192.168.100.8",
00:07:18.914      "adrfam": "ipv4",
00:07:18.914      "trsvcid": "4420",
00:07:18.914      "subnqn": "nqn.2016-06.io.spdk:cnode2",
00:07:18.914      "hostnqn": "nqn.2016-06.io.spdk:host2",
00:07:18.914      "hdgst": false,
00:07:18.914      "ddgst": false
00:07:18.914    },
00:07:18.914    "method": "bdev_nvme_attach_controller"
00:07:18.914  },{
00:07:18.914    "params": {
00:07:18.914      "name": "Nvme3",
00:07:18.914      "trtype": "rdma",
00:07:18.914      "traddr": "192.168.100.8",
00:07:18.914      "adrfam": "ipv4",
00:07:18.914      "trsvcid": "4420",
00:07:18.914      "subnqn": "nqn.2016-06.io.spdk:cnode3",
00:07:18.914      "hostnqn": "nqn.2016-06.io.spdk:host3",
00:07:18.914      "hdgst": false,
00:07:18.914      "ddgst": false
00:07:18.914    },
00:07:18.914    "method": "bdev_nvme_attach_controller"
00:07:18.914  },{
00:07:18.914    "params": {
00:07:18.914      "name": "Nvme4",
00:07:18.914      "trtype": "rdma",
00:07:18.914      "traddr": "192.168.100.8",
00:07:18.914      "adrfam": "ipv4",
00:07:18.914      "trsvcid": "4420",
00:07:18.914      "subnqn": "nqn.2016-06.io.spdk:cnode4",
00:07:18.914      "hostnqn": "nqn.2016-06.io.spdk:host4",
00:07:18.914      "hdgst": false,
00:07:18.914      "ddgst": false
00:07:18.914    },
00:07:18.914    "method": "bdev_nvme_attach_controller"
00:07:18.914  },{
00:07:18.914    "params": {
00:07:18.914      "name": "Nvme5",
00:07:18.914      "trtype": "rdma",
00:07:18.914      "traddr": "192.168.100.8",
00:07:18.914      "adrfam": "ipv4",
00:07:18.914      "trsvcid": "4420",
00:07:18.914      "subnqn": "nqn.2016-06.io.spdk:cnode5",
00:07:18.914      "hostnqn": "nqn.2016-06.io.spdk:host5",
00:07:18.914      "hdgst": false,
00:07:18.914      "ddgst": false
00:07:18.914    },
00:07:18.914    "method": "bdev_nvme_attach_controller"
00:07:18.914  },{
00:07:18.914    "params": {
00:07:18.914      "name": "Nvme6",
00:07:18.914      "trtype": "rdma",
00:07:18.914      "traddr": "192.168.100.8",
00:07:18.914      "adrfam": "ipv4",
00:07:18.914      "trsvcid": "4420",
00:07:18.914      "subnqn": "nqn.2016-06.io.spdk:cnode6",
00:07:18.914      "hostnqn": "nqn.2016-06.io.spdk:host6",
00:07:18.914      "hdgst": false,
00:07:18.914      "ddgst": false
00:07:18.914    },
00:07:18.914    "method": "bdev_nvme_attach_controller"
00:07:18.914  },{
00:07:18.914    "params": {
00:07:18.914      "name": "Nvme7",
00:07:18.914      "trtype": "rdma",
00:07:18.914      "traddr": "192.168.100.8",
00:07:18.914      "adrfam": "ipv4",
00:07:18.914      "trsvcid": "4420",
00:07:18.914      "subnqn": "nqn.2016-06.io.spdk:cnode7",
00:07:18.914      "hostnqn": "nqn.2016-06.io.spdk:host7",
00:07:18.914      "hdgst": false,
00:07:18.914      "ddgst": false
00:07:18.914    },
00:07:18.914    "method": "bdev_nvme_attach_controller"
00:07:18.914  },{
00:07:18.914    "params": {
00:07:18.914      "name": "Nvme8",
00:07:18.914      "trtype": "rdma",
00:07:18.914      "traddr": "192.168.100.8",
00:07:18.914      "adrfam": "ipv4",
00:07:18.914      "trsvcid": "4420",
00:07:18.914      "subnqn": "nqn.2016-06.io.spdk:cnode8",
00:07:18.914      "hostnqn": "nqn.2016-06.io.spdk:host8",
00:07:18.914      "hdgst": false,
00:07:18.914      "ddgst": false
00:07:18.914    },
00:07:18.914    "method": "bdev_nvme_attach_controller"
00:07:18.914  },{
00:07:18.914    "params": {
00:07:18.914      "name": "Nvme9",
00:07:18.914      "trtype": "rdma",
00:07:18.914      "traddr": "192.168.100.8",
00:07:18.914      "adrfam": "ipv4",
00:07:18.914      "trsvcid": "4420",
00:07:18.914      "subnqn": "nqn.2016-06.io.spdk:cnode9",
00:07:18.914      "hostnqn": "nqn.2016-06.io.spdk:host9",
00:07:18.914      "hdgst": false,
00:07:18.914      "ddgst": false
00:07:18.914    },
00:07:18.914    "method": "bdev_nvme_attach_controller"
00:07:18.914  },{
00:07:18.914    "params": {
00:07:18.914      "name": "Nvme10",
00:07:18.914      "trtype": "rdma",
00:07:18.914      "traddr": "192.168.100.8",
00:07:18.914      "adrfam": "ipv4",
00:07:18.914      "trsvcid": "4420",
00:07:18.914      "subnqn": "nqn.2016-06.io.spdk:cnode10",
00:07:18.914      "hostnqn": "nqn.2016-06.io.spdk:host10",
00:07:18.914      "hdgst": false,
00:07:18.914      "ddgst": false
00:07:18.914    },
00:07:18.914    "method": "bdev_nvme_attach_controller"
00:07:18.914  }'
00:07:18.917  [2021-06-07 12:09:25.693793] Starting SPDK v21.07-pre git sha1 c60e0687c / DPDK 21.02.0 initialization...
00:07:18.917  [2021-06-07 12:09:25.693894] [ DPDK EAL parameters: [2021-06-07 12:09:25.693910] bdevperf [2021-06-07 12:09:25.693921] --no-shconf [2021-06-07 12:09:25.693932] -c 0x1 [2021-06-07 12:09:25.693942] --log-level=lib.eal:6 [2021-06-07 12:09:25.693952] --log-level=lib.cryptodev:5 [2021-06-07 12:09:25.693963] --log-level=user1:6 [2021-06-07 12:09:25.693974] --base-virtaddr=0x200000000000 [2021-06-07 12:09:25.693984] --match-allocations [2021-06-07 12:09:25.693995] --file-prefix=spdk_pid473454 [2021-06-07 12:09:25.694006] ]
00:07:18.966  EAL: No free 2048 kB hugepages reported on node 1
00:07:18.966  EAL: No available 1048576 kB hugepages reported
00:07:18.970  EAL: No legacy callbacks, legacy socket not created
00:07:19.008  [2021-06-07 12:09:25.784878] app.c: 537:spdk_app_start: *NOTICE*: Total cores available: 1
00:07:19.233  [2021-06-07 12:09:26.009800] reactor.c: 946:reactor_run: *NOTICE*: Reactor started on core 0
00:07:19.242  [2021-06-07 12:09:26.019303] accel_engine.c: 853:spdk_accel_engine_initialize: *NOTICE*: Accel engine initialized to use software engine.
00:07:19.631  [2021-06-07 12:09:26.407635] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.631  [2021-06-07 12:09:26.407817] blobstore.c:3054:bs_channel_create: *ERROR*: Failed to create device channel.
00:07:19.632  [2021-06-07 12:09:26.407877] blobstore.c:5310:bs_register_md_thread: *ERROR*: Failed to get IO channel.
00:07:19.632  [2021-06-07 12:09:26.407949] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.632  [2021-06-07 12:09:26.408008] vbdev_gpt.c: 500:vbdev_gpt_read_gpt: *ERROR*: Failed to get an io_channel.
00:07:19.633  [2021-06-07 12:09:26.408066] vbdev_gpt.c: 553:vbdev_gpt_examine: *ERROR*: Failed to read info from bdev Nvme6n1
00:07:19.672  [2021-06-07 12:09:26.448248] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.672  [2021-06-07 12:09:26.448342] blobstore.c:3054:bs_channel_create: *ERROR*: Failed to create device channel.
00:07:19.672  [2021-06-07 12:09:26.448403] blobstore.c:5310:bs_register_md_thread: *ERROR*: Failed to get IO channel.
00:07:19.672  [2021-06-07 12:09:26.448473] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.673  [2021-06-07 12:09:26.448531] vbdev_gpt.c: 500:vbdev_gpt_read_gpt: *ERROR*: Failed to get an io_channel.
00:07:19.673  [2021-06-07 12:09:26.448585] vbdev_gpt.c: 553:vbdev_gpt_examine: *ERROR*: Failed to read info from bdev Nvme7n1
00:07:19.712  [2021-06-07 12:09:26.488445] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.712  [2021-06-07 12:09:26.488528] blobstore.c:3054:bs_channel_create: *ERROR*: Failed to create device channel.
00:07:19.712  [2021-06-07 12:09:26.488583] blobstore.c:5310:bs_register_md_thread: *ERROR*: Failed to get IO channel.
00:07:19.713  [2021-06-07 12:09:26.488655] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.713  [2021-06-07 12:09:26.488713] vbdev_gpt.c: 500:vbdev_gpt_read_gpt: *ERROR*: Failed to get an io_channel.
00:07:19.713  [2021-06-07 12:09:26.488769] vbdev_gpt.c: 553:vbdev_gpt_examine: *ERROR*: Failed to read info from bdev Nvme8n1
00:07:19.752  [2021-06-07 12:09:26.528522] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.752  [2021-06-07 12:09:26.528603] blobstore.c:3054:bs_channel_create: *ERROR*: Failed to create device channel.
00:07:19.752  [2021-06-07 12:09:26.528662] blobstore.c:5310:bs_register_md_thread: *ERROR*: Failed to get IO channel.
00:07:19.753  [2021-06-07 12:09:26.528733] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.753  [2021-06-07 12:09:26.528792] vbdev_gpt.c: 500:vbdev_gpt_read_gpt: *ERROR*: Failed to get an io_channel.
00:07:19.753  [2021-06-07 12:09:26.528849] vbdev_gpt.c: 553:vbdev_gpt_examine: *ERROR*: Failed to read info from bdev Nvme9n1
00:07:19.792  [2021-06-07 12:09:26.568508] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.792  [2021-06-07 12:09:26.568589] blobstore.c:3054:bs_channel_create: *ERROR*: Failed to create device channel.
00:07:19.792  [2021-06-07 12:09:26.568646] blobstore.c:5310:bs_register_md_thread: *ERROR*: Failed to get IO channel.
00:07:19.793  [2021-06-07 12:09:26.568715] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.793  [2021-06-07 12:09:26.568772] vbdev_gpt.c: 500:vbdev_gpt_read_gpt: *ERROR*: Failed to get an io_channel.
00:07:19.793  [2021-06-07 12:09:26.568830] vbdev_gpt.c: 553:vbdev_gpt_examine: *ERROR*: Failed to read info from bdev Nvme10n1
00:07:19.822  [2021-06-07 12:09:26.598387] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.822  [2021-06-07 12:09:26.598611] bdevperf.c:1189:_bdevperf_construct_job: *ERROR*: Could not get io_channel for device Nvme1n1, error=0
00:07:19.822  [2021-06-07 12:09:26.598682] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.822  [2021-06-07 12:09:26.598700] bdevperf.c:1189:_bdevperf_construct_job: *ERROR*: Could not get io_channel for device Nvme2n1, error=0
00:07:19.822  [2021-06-07 12:09:26.598720] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.822  [2021-06-07 12:09:26.598733] bdevperf.c:1189:_bdevperf_construct_job: *ERROR*: Could not get io_channel for device Nvme3n1, error=0
00:07:19.823  [2021-06-07 12:09:26.598752] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.823  [2021-06-07 12:09:26.598766] bdevperf.c:1189:_bdevperf_construct_job: *ERROR*: Could not get io_channel for device Nvme4n1, error=0
00:07:19.823  [2021-06-07 12:09:26.598785] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.823  [2021-06-07 12:09:26.598798] bdevperf.c:1189:_bdevperf_construct_job: *ERROR*: Could not get io_channel for device Nvme5n1, error=0
00:07:19.823  [2021-06-07 12:09:26.598816] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.823  [2021-06-07 12:09:26.598829] bdevperf.c:1189:_bdevperf_construct_job: *ERROR*: Could not get io_channel for device Nvme6n1, error=0
00:07:19.823  [2021-06-07 12:09:26.598847] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.823  [2021-06-07 12:09:26.598860] bdevperf.c:1189:_bdevperf_construct_job: *ERROR*: Could not get io_channel for device Nvme7n1, error=0
00:07:19.824  [2021-06-07 12:09:26.598877] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.824  [2021-06-07 12:09:26.598890] bdevperf.c:1189:_bdevperf_construct_job: *ERROR*: Could not get io_channel for device Nvme8n1, error=0
00:07:19.824  [2021-06-07 12:09:26.598921] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.824  [2021-06-07 12:09:26.598936] bdevperf.c:1189:_bdevperf_construct_job: *ERROR*: Could not get io_channel for device Nvme9n1, error=0
00:07:19.824  [2021-06-07 12:09:26.598954] thread.c:2020:spdk_get_io_channel: *ERROR*: could not find io_device 0x7fc182b84e40
00:07:19.824  [2021-06-07 12:09:26.598968] bdevperf.c:1189:_bdevperf_construct_job: *ERROR*: Could not get io_channel for device Nvme10n1, error=0
00:07:19.832  Job run time less than one microsecond, no performance data will be shown
00:07:19.832  
 Job: Nvme1n1 (Core Mask 0x1)
00:07:19.832  	 Verification LBA range: start 0x0 length 0x400
00:07:19.832  	 Nvme1n1             :       -nan IOPS       -nan MiB/s
00:07:19.832  	                     :       -nan Fail/s     -nan TO/s
00:07:19.832  
 Job: Nvme2n1 (Core Mask 0x1)
00:07:19.832  	 Verification LBA range: start 0x0 length 0x400
00:07:19.832  	 Nvme2n1             :       -nan IOPS       -nan MiB/s
00:07:19.832  	                     :       -nan Fail/s     -nan TO/s
00:07:19.832  
 Job: Nvme3n1 (Core Mask 0x1)
00:07:19.832  	 Verification LBA range: start 0x0 length 0x400
00:07:19.832  	 Nvme3n1             :       -nan IOPS       -nan MiB/s
00:07:19.832  	                     :       -nan Fail/s     -nan TO/s
00:07:19.832  
 Job: Nvme4n1 (Core Mask 0x1)
00:07:19.832  	 Verification LBA range: start 0x0 length 0x400
00:07:19.832  	 Nvme4n1             :       -nan IOPS       -nan MiB/s
00:07:19.832  	                     :       -nan Fail/s     -nan TO/s
00:07:19.832  
 Job: Nvme5n1 (Core Mask 0x1)
00:07:19.832  	 Verification LBA range: start 0x0 length 0x400
00:07:19.832  	 Nvme5n1             :       -nan IOPS       -nan MiB/s
00:07:19.832  	                     :       -nan Fail/s     -nan TO/s
00:07:19.832  
 Job: Nvme6n1 (Core Mask 0x1)
00:07:19.832  	 Verification LBA range: start 0x0 length 0x400
00:07:19.832  	 Nvme6n1             :       -nan IOPS       -nan MiB/s
00:07:19.832  	                     :       -nan Fail/s     -nan TO/s
00:07:19.832  
 Job: Nvme7n1 (Core Mask 0x1)
00:07:19.832  	 Verification LBA range: start 0x0 length 0x400
00:07:19.832  	 Nvme7n1             :       -nan IOPS       -nan MiB/s
00:07:19.832  	                     :       -nan Fail/s     -nan TO/s
00:07:19.832  
 Job: Nvme8n1 (Core Mask 0x1)
00:07:19.832  	 Verification LBA range: start 0x0 length 0x400
00:07:19.832  	 Nvme8n1             :       -nan IOPS       -nan MiB/s
00:07:19.832  	                     :       -nan Fail/s     -nan TO/s
00:07:19.832  
 Job: Nvme9n1 (Core Mask 0x1)
00:07:19.832  	 Verification LBA range: start 0x0 length 0x400
00:07:19.832  	 Nvme9n1             :       -nan IOPS       -nan MiB/s
00:07:19.832  	                     :       -nan Fail/s     -nan TO/s
00:07:19.832  
 Job: Nvme10n1 (Core Mask 0x1)
00:07:19.832  	 Verification LBA range: start 0x0 length 0x400
00:07:19.832  	 Nvme10n1            :       -nan IOPS       -nan MiB/s
00:07:19.832  	                     :       -nan Fail/s     -nan TO/s
00:07:19.832  
 =============================================================
00:07:19.832  
 Total                       :       -nan IOPS       -nan MiB/s
00:07:19.832  
                             :       -nan Fail/s     -nan TO/s
00:07:19.832  [2021-06-07 12:09:26.608473] app.c: 628:spdk_app_stop: *WARNING*: spdk_app_stop'd on non-zero
00:16:05.199  Build timed out (after 15 minutes). Marking the build as failed.
```

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_49182.html

**Comment by jimharris** [comment]
This is most likely due to recently merged commit 2246a9371809c30333b1844afbf9772c4b06.

I've pushed a revert for now: https://review.spdk.io/gerrit/c/spdk/spdk/+/8199

**Comment by jimharris** [comment]
Patch merged.  Closing this issue.

We will keep #1977 open for now to track this general io_device lookup issue, should the revert not fix it.
